libpod/util_test.go

```diff
@@ -0,0 +1,19 @@
+package libpod
+
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+)
+var (
+	sliceData = []string{"one", "two", "three", "four"}
+)
+
+func TestStringInSlice(t *testing.T) {
+	// string is in the slice
+	assert.True(t, StringInSlice("one", sliceData))
+	// string is not in the slice
+	assert.False(t, StringInSlice("five", sliceData))
+	// string is not in empty slice
+	assert.False(t, StringInSlice("one", []string{}))
+}
```

Comment:
Add line

Reply:
@rhatdan are you wanting a newline? if so, gofmt keeps yanking it out.